lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -597,7 +597,13 @@ def generate_clone_spec(src_config)
 
     if get_config(:customization_vlan)
       vlan_list = get_config(:customization_vlan).split(',')
-      networks = vlan_list.map { |vlan| find_network(vlan) }
+      networks =
+        if get_config(:customization_sw_uuid)
+          sw_uuid = get_config(:customization_sw_uuid)
```

Comment:
I'll confess the `customization_sw_uuid` feature isn't fresh in my mind as I don't use it. It seems to be able to take the value of `auto`, do you think that'll cause problems if people try to use it at the same time as this feature? (or rather, do you think people will try?)